egs/hkust/s5/local/chain/run_tdnn.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
```

Comment:
For the chain and nnet3 scripts, if you've copied them from another directory/eg, please mention the source on the top -- that would help tracking of changes (for example propagating changes when the originating script changes).